### PR TITLE
Added support for mongo uri

### PR DESCRIPTION
**Pull request by qqmbr4k**
Removed env vars MONGO_HOST, MONGO_PORT. Added env var MONGO_URI

### CODE REVIEW

**Comment by andreyshikov** [comment]
@pferrel tested with atlas, ready to merge

**Comment by pferrel** [comment]
@andreyshikov This should be against `develop`

This requires changes to `harness-env` and perhaps the `harness-start` script to print the config for mongo that the server is using when started.

I do not see the env changes here.

**Comment by pferrel** [comment]
Also we need docs for how to set the 2 users we need for Atlas.

File: rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoConfig.scala [hunk 1]
```diff
@@ -17,16 +17,14 @@
...
+case class MongoConfig(private val uriString: String) {
+  val uri = new MongoClientURI(uriString)
+}
 
 object MongoConfig {
-  private lazy val config = ConfigFactory.load
 
-  lazy val mongo = config.as[MongoConfig]("mongo")
-}
+  val mongo = MongoConfig(sys.env.getOrElse("MONGO_URI", throw new RuntimeException("Environment variable MONGO_URI must be set")))
```
Comment (andreyshikov) [inline rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoConfig.scala:29]: @pferrel am I right here?

Comment (pferrel) [reply rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoConfig.scala:29]: Looks correct, as long as we always set the value in script or code. If this exception is thrown it means someone has removed the setting.

**Comment by pferrel** [comment]
this should be merged with `develop`, not the release branch.

@andreyshikov Will merge this branch by hand, not in the github gui.

**Pull request closed without merging**